Documentation-Requirements.md

```diff
@@ -79,3 +80,17 @@ utility.
       ```
 
   ```
+
+# Images
+
+All binary images files must be in standard and well supported formats, such
+as JPEG and PNG.
```

Comment:
Maybe we can note that PNG is preferable for diagrams. JPG is not good for vector style graphics generally (but good for photos).

Reply:
Good call - branch updated.